doc/api/events.md

```diff
@@ -383,6 +383,17 @@ Installing a listener using this symbol does not change the behavior once an
 `'error'` event is emitted, therefore the process will still crash if no
 regular `'error'` listener is installed.
 
+### `EventEmitter.setEventTargetMaxListeners(n[, ...eventTargets])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `n` {number} A non-negative number. The maximum number of listeners per
```

Comment:
It would make sense for this to work with both EventEmitters and EventTargets and should be pretty easy to do right?

Reply:
Doesn't have to happen in this PR or block it I don't mind following up with an improvement.